config/constants.js

```diff
@@ -0,0 +1,53 @@
+module.exports = {
+  connectDB: {
+    success: 'db is connected successfully',
+    failure: 'db could not be connected',
+  },
+  labels: {
+    githubPassportStrategy: {
+      githubProfile: {
+        failure: 'Github profile.id is not found!',
+      },
+      findingUser: {
+        error: 'Error while finding the user if its already existed in db (github)',
+        found: 'User is existed in db (github)',
+      },
+      savingNewUser: {
+        error: 'Error in saving user through (github strategy)',
+        success: 'New user is saved in db through (github strategy)',
+        failure: 'New user could not be saved in db through (github strategy)',
```

Comment:
What is the difference between error and failure? Same comment as line 13

Reply:
Resolved the issue